src/v2/models/FormModel.js

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the 'License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+  'okta',
+  './BaseLoginModel'
+],
+function (Okta, BaseLoginModel) {
+  return BaseLoginModel.extend({
+    initialize: function (options) {
+      this.options = options || {};
+      this.appState = this.options.appState;
```

Comment:
- nit: doesn't seem necessary to reassign `appState` to `this.`
- if we still call it `appState`, shall we rename the `WidgetState` to 'AppState`?